### PR TITLE
Fix for safari/ios not being able to access filereader

### PR DESCRIPTION
**Pull request by seripap**
Should be able to upload files through iOS now

### CODE REVIEW

File: src/js/window.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default class WindowHandler {
     if (this.fileHandler.isSupported) {
       $('#send-file').click((e) => {
         e.preventDefault();
-        $('#fileInput').trigger('click');
+        document.getElementById('fileInput').click();
```
Comment (alanfriedman) [inline src/js/window.js]: Getting rid of jQuery?

Comment (alanfriedman) [inline src/js/window.js]: Oh, you may be looking for `$('#fileInput')[0].trigger('click');`

Comment (seripap) [inline src/js/window.js]: Just testing different ways to attempt to trigger the file input on safari, we can actually revert it back.

**Comment by alanfriedman** [comment]
Does this work? IIRC, Safari has some pretty strict limits on how you can trigger user actions. Usually there has to be a direct user click/touch on a link/button.

**Comment by seripap** [comment]
[Of course it works](https:/darkwire.herokuapp.com)

**Comment by alanfriedman** [comment]
Doesn't work on iOS.

**Comment by seripap** [comment]
My bad- that was the wrong branch deployed on heroku. Try it now.

**Comment by alanfriedman** [comment]
Ok it works now.

**Pull request merged**